packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -183,13 +185,22 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
             style,
             testId,
             dropdownStyle,
+            // the following props are being included here to avoid
+            // passing them down to the opener as part of sharedProps
+            /* eslint-disable no-unused-vars */
+            children,
+            onChange,
+            selectedValues,
```

Comment:
Oh that's right! thanks :)

Reply:
Just removed those lines and `flow` threw this error:

```
React$Element<class ActionMenuOpener>
Cannot create `ActionMenuOpener` element because property `onChange` is missing in  `Props` [1] but exists in  props [2].Flow(InferError)
action-menu-opener.js(40, 63): [1] `Props`
action-menu.js(196, 13): [2] props
Cannot create `ActionMenuOpener` element because property `selectedValues` is missing in  `Props` [1] but exists in  props [2].Flow(InferError)
action-menu-opener.js(40, 63): [1] `Props`
action-menu.js(196, 13): [2] props
```